tasty-query/shared/src/main/scala/tastyquery/Types.scala

```diff
@@ -198,6 +198,24 @@ object Types {
     final def select(name: TypeName)(using Context): TypeRef =
       TypeRef(this, name)
 
+    final def lookupMember(name: Name)(using Context): Option[NamedType] =
+      resolveMember(name, this) match
+        case ResolveMemberResult.NotFound =>
+          None
+        case resolved: ResolveMemberResult.TermMember =>
+          Some(TermRef.fromResolved(this, resolved))
+        case resolved: ResolveMemberResult.ClassMember =>
+          Some(TypeRef.fromResolved(this, resolved))
+        case resolved: ResolveMemberResult.TypeMember =>
+          Some(TypeRef.fromResolved(this, name.asInstanceOf[TypeName], resolved))
+    end lookupMember
+
+    final def lookupMember(name: TermName)(using Context): Option[TermRef] =
```

Comment:
one question: what should happen when you call this on a `PackageRef`, and its the name of a package?

Reply:
Ok it produces `ResolveMemberResult.NotFound`